README.md

```diff
@@ -1,3 +1,5 @@
+[![Travis](https://img.shields.io/travis/rust-lang/rust.svg?style=plastic)](https://github.com/systers/vms)
+
```

Comment:
👋 @Janiceilene @MeepyMay, we had one, there are two presently, you may want to remove the existing one if you plan to push the same throughout.

Reply:
Oh no!  I'll delete this one.  I completely missed the others.